README.md

```diff
@@ -66,6 +66,33 @@ By definition, members work on their respective squad, although they are free to
 
 ## 4. Release process
 
+### Release engineer as a role
+
+  The release engineer responsibilities are, but not limited to:
+
+  1. Own the entire release process (step-by-step).
+  2. Provide visibility, at all stages, of the release to the wider audience (e.g. squad, tribe and iOS chapter lead).
+  3. Collaborate with QA by:
+    1. Providing visibility to potential blockers to the wider audience.
+    2. Escalating abnormal influx of bugs, so the release, as a whole, can be reassessed.
+  4. Making sure he/she can dedicate enough time for the release. In case this is not possible (due to other squad commitments), please inform the iOS chapter lead.
```

Comment:
AFAIK we are now consolidating to the singular `they` for gender-neutral pronouns. So this might become `4. Making sure they can dedicate enough time for the release.`